src/Components/FilterSection/CitizenshipPopover.tsx

```diff
@@ -39,49 +41,124 @@ const CitizenshipPopover = ({
+        filter: {
+          id: 1,
+          columnField: 'citizenship',
+          operatorValue: 'customCitizenshipOperator',
+          value: ['citizen'],
+        },
+      });
+    }
 
+    //update citizenshipFilterIsChecked state
     setCitizenshipFilterIsChecked(updatedCitizenshipFilterIsChecked);
   };
 
   const typedCitizenshipFilterIsChecked = Object.keys(citizenshipFilterIsChecked) as CitizenLabels[];
-  const citizenshipCheckboxFilters = typedCitizenshipFilterIsChecked.map((citizenshipType) => {
-    return (
-      <FormControlLabel
-        key={citizenshipType}
-        className="popover"
-        label={citizenshipFilterFormControlLabels[citizenshipType]}
-        control={
-          <Checkbox
-            checked={citizenshipFilterIsChecked[citizenshipType]}
-            onChange={() => handleFilterSelect(citizenshipType)}
+
+  const renderCitizenshipFilters = (citizenshipFilters: Record<CitizenLabels, boolean>) => {
+    if (citizenshipFilters.green_card) {
+      const allCitizenshipCheckboxes = typedCitizenshipFilterIsChecked.map((citizenshipType) => {
+        //here we need to add an sx prop to indent them if they're the gc_filters
+        const isGreenCardSubCitizenshipType =
+          citizenshipType === 'gc_5plus' ||
+          citizenshipType === 'gc_18plus_no5' ||
+          citizenshipType === 'gc_under18_no5' ||
+          citizenshipType === 'gc_under19_pregnant_no5';
+
+        return (
+          <FormControlLabel
+            key={citizenshipType}
+            className={isGreenCardSubCitizenshipType ? 'gc-subcitizen-indentation' : ''}
+            label={citizenshipFilterFormControlLabels[citizenshipType]}
+            control={
+              <Checkbox
+                checked={citizenshipFilters[citizenshipType]}
+                onChange={() => handleFilterSelect(citizenshipType)}
+              />
+            }
+          />
+        );
+      });
+
+      return allCitizenshipCheckboxes;
+    } else {
+      //green_card is false
+      const initialThreeFilters = ['non_citizen', 'green_card', 'refugee'] as CitizenLabels[];
+      const initialCitizenshipCheckboxes = initialThreeFilters.map((initialFilter) => {
+        return (
+          <FormControlLabel
+            key={initialFilter}
+            label={citizenshipFilterFormControlLabels[initialFilter]}
+            control={
+              <Checkbox
+                checked={citizenshipFilters[initialFilter]}
+                onChange={() => handleFilterSelect(initialFilter)}
+              />
+            }
           />
-        }
-      />
-    );
-  });
+        );
+      });
+
+      return initialCitizenshipCheckboxes;
+    }
+  };
 
-  return <Stack>{citizenshipCheckboxFilters}</Stack>;
+  return (
+    <Stack sx={{ padding: '0.5rem' }}>
+      <Stack sx={{ color: '#000000', fontWeight: 500, mt: '.5rem', ml: '.5rem' }}>
+        <FormattedMessage id="citizenshipPopover.showBenefits" defaultMessage="Show benefits available to:" />
```

Comment:
This will require adding the requested text changes in our new translations UI via api/translations/admin.